test/functional/apps/dashboard_elements/controls/control_group_chaining.ts

```diff
@@ -44,7 +43,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       /* start by adding some incomplete data so that we can test `exists` query */
       await common.navigateToApp('console');
-      await console.collapseHelp();
+      await console.closeHelpIfExists();
```

Comment:
This has nothing to do with the actual fix but it was causing issues when I was running these teest locally - `collapseHelp` assumes that the console help is always present so switching to `closeHelpIfExists` makes it more reliable.